src/lib/Input.tsx

```diff
@@ -56,6 +72,9 @@ const Input = <T extends FieldValues>(props: InputProps<T>) => {
   if (step && type !== "number" && type !== "range") {
     throw new Error("step can only be used with number or range inputs");
   }
+  if ((minLength || maxLength) && type && !["text", "password", "textarea", "search", "tel", "url", "email"].includes(type)) {
```

Comment:
@Michele-Masciave Passing 0 will throw an exception :) 

I would check also that values are > 0 :)

Reply:
Done as suggested